docs: generate typescript data models (#432)
asyncapi/modelina: docs/usage.md

TODO 

## Generate TypeScript models

> Among many of the output languages we support, TypeScript is one of them. You can check out the [TypeScript documentation for more advanced use-cases](./languages/TypeScript.md).
> Check out this [example out for a basic live demonstration](../examples/generate-typescript-models).

Reviewer: ```suggestion
TypeScript is one of the many output languages we support. Check out the following [TypeScript documentation for more advanced use-cases](./languages/TypeScript.md).

Or, take a look at this [basic example for a live demonstration](../examples/generate-typescript-models).
```

TODO 

## Generate TypeScript models

> There are one way to generate models from object.
> 
> - [Generate Data Model](../examples/generate-typescript-models)

Reviewer 2: Maybe something along these lines is better:

```suggestion
Among many of the output languages we support, TypeScript is one of them. You can check out the [TypeScript documentation for more advanced use-cases](./languages/TypeScript.md).

Check out this [example out for a basic live demonstration](../examples/generate-typescript-models).
```

Thoughts @md-saif-husain ?

Author: @jonaslagoni : Thanks for feedback . Above suggestion make sense . I will update the usage.md

Author: @jonaslagoni : I have updated usage.md as per your suggestion
